Engine/Objects/Actor.cpp

```diff
@@ -3984,13 +3984,13 @@ bool CheckActors_proximity() {
     int for_z;            // [sp+10h] [bp-Ch]@5
 
     distance = 5120;
-    if (uCurrentlyLoadedLevelType == LEVEL_Indoor) distance = 2560;
```

Comment:
There is a still about 128 difference between indicators and this distance. Also I think double less distance in dungeons makes sense as dungeons generally more twisted and have more walls than outdoor. Maybe we should also make line of sight checks? Because now it seems we will have less possibilities to rest in dungeons and it will affect some invisibility sneak strats.

Reply:
I will revert. This behaviour is as vanilla.